src/Esfa.Vacancy.Register.UnitTests/SearchApprenticeship/Application/GivenASearchApprenticeshipVacanciesRequestValidator/WhenValidatingPostedInDays.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.Collections.Generic;
+using Esfa.Vacancy.Register.Application.Queries.SearchApprenticeshipVacancies;
+using Esfa.Vacancy.Register.Domain.Validation;
+using FluentAssertions;
+using FluentValidation.Results;
+using NUnit.Framework;
+
+namespace Esfa.Vacancy.Register.UnitTests.SearchApprenticeship.Application.GivenASearchApprenticeshipVacanciesRequestValidator
+{
+    [TestFixture]
+    public class WhenValidatingPostedInDays
+    {
+        private SearchApprenticeshipVacanciesRequestValidator _validator;
+
+        [SetUp]
+        public void Setup()
+        {
+            _validator = new SearchApprenticeshipVacanciesRequestValidator();
+        }
+
+        [TestCaseSource(nameof(TestCases))]
```

Comment:
Maybe stick the test cases near the top of the file.

Reply:
I've only ever seen the test cases below the tests so that its easy to find the tests.

Reply:
Fair enough its just if it becomes a long class its a bit difficult if you have to scroll to the bottom. Bu t the same could be said the other way scrolling to the top.